src/ui/components/filters/filterboxcomponent.js

```diff
@@ -161,6 +161,14 @@ export default class FilterBoxComponent extends Component {
         });
       });
     }
+
```

Comment:
would it be possible to do this in the above loop?

Reply:
Ah yes, I can refactor that loop so that it works for both!